### PR TITLE
ohybridproxy: fix up Makefile

### PR DESCRIPTION
**Pull request by neheb**
Use a more standard setup for local tarballs to clean up the Makefile.

Fixed license information.

Added PKG_BUILD_PARALLEL for faster compilation.

Removed upstreamed patch.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: ohybridproxy/Makefile [hunk 1]
```diff
@@ -8,16 +8,18 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ohybridproxy
 PKG_SOURCE_VERSION:=0dfef1eb5f067250a5f24a899536879ea4fdc4c5
-PKG_VERSION:=2020-05-22-$(PKG_SOURCE_VERSION)
-PKG_RELEASE:=1
+PKG_SOURCE_DATE:=2020-05-22
```
Comment (adschm) [inline ohybridproxy/Makefile:11]: Don't we typically group the SOURCE_DATE and SOURCE_VERSION together with the other PKG_SOURCE_* variables?

Comment (adschm) [reply ohybridproxy/Makefile:11]: Oops, this is openwrt-routing, sorry if I invaded somebody else's realm or judgement here ...

Comment (neheb) [reply ohybridproxy/Makefile:11]: I do this to keep each section small.

It makes sense to put date and version at the top anyway.

**Comment by neheb** [comment]
ping @mwarning

**Comment by mwarning** [comment]
Thanks.

**Pull request merged**